build.sbt

```diff
@@ -4,8 +4,8 @@ version := "0.1"
 
 scalaVersion := "2.11.12"
 
-libraryDependencies += "org.apache.spark" %% "spark-streaming" % "2.2.2"
-libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.2.2"
+libraryDependencies += "org.apache.spark" %% "spark-streaming" % "2.2.2" % "provided"
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.2.2" % "provided"
```

Comment:
which version of spark is provided on the cluster?

Reply:
Spark 2.2.0

Reply:
do you see any issue here?

Reply:
Issues related to version ?

Reply:
the code is build agains `2.2.2` but you ruining in on `2.2.0`. this is the issue.